frontend/app/src/components/defi/wizard/ModuleSelector.vue

```diff
@@ -1,145 +1,163 @@
+const { start: fetch } = useTimeoutFn(() => resetStatus(), 800, {
+  immediate: false
+});
+const { start: clearNfBalances } = useTimeoutFn(
+  () => balancesStore.$reset(),
+  800,
+  { immediate: false }
+);
 
-const update = async (activeModules: Module[], clearSearch = true) => {
-  if (clearSearch) {
-    set(search, '');
-    setTimeout(() => {
-      const searchField = get(autocomplete) as any;
-      if (searchField) {
-        searchField.focus();
-      }
-    }, 10);
-  }
+const update = async (activeModules: Module[]) => {
   set(loading, true);
-
   await updateSettings({ activeModules });
-  onModuleActivation(activeModules);
-  set(selectedModules, activeModules);
   set(loading, false);
 };
 
-const unselect = async (identifier: Module) => {
-  const selected = get(selectedModules);
-  const previouslyActive = [...selected];
-  const selectionIndex = selected.indexOf(identifier);
-  if (selectionIndex < 0) {
-    return;
+const switchModule = async (module: Module, enabled: boolean) => {
+  const active = get(activeModules);
+  let modules: Module[];
+  if (enabled) {
+    modules = [...active, module];
+  } else {
+    modules = active.filter(m => m !== module);
   }
-  selected.splice(selectionIndex, 1);
-  await update(selected, false);
 
-  if (wasDeactivated(selected, previouslyActive, Module.NFTS)) {
-    clearNfBalances();
+  await update(modules);
+  if (module === Module.NFTS) {
+    if (enabled) {
+      fetch();
+    } else {
+      clearNfBalances();
+    }
   }
 };
 
-onMounted(() => {
-  set(selectedModules, get(activeModules));
-});
+const enableAll = async () => {
+  const allModules = supportedModules.map(x => x.identifier);
+  const active = get(activeModules);
+  const activatedModules = allModules.filter(m => !active.includes(m));
+  await update(allModules);
 
-const { t } = useI18n();
+  if (activatedModules.includes(Module.NFTS)) {
+    fetch();
+  }
+};
+
+const disableAll = async () => {
+  const active = get(activeModules);
+  await update([]);
+  if (active.includes(Module.NFTS)) {
+    clearNfBalances();
+  }
+};
 </script>
 
 <template>
-  <v-autocomplete
-    ref="autocomplete"
-    :value="selectedModules"
-    :search-input.sync="search"
-    :items="supportedModules"
-    hide-details
-    hide-selected
-    hide-no-data
-    clearables
-    multiple
-    outlined
-    chips
-    :disabled="loading"
-    :loading="loading"
-    :open-on-clear="false"
-    :label="t('module_selector.label')"
-    item-text="name"
-    item-value="identifier"
-    class="module-selector"
-    @input="update($event)"
-  >
-    <template #selection="data">
-      <v-chip
-        :id="`defi-module-${data.item.identifier}`"
-        close
-        pill
-        @click:close="unselect(data.item.identifier)"
-      >
-        <v-avatar left class="d-flex">
-          <adaptive-wrapper
-            class="d-flex align-center"
-            width="100%"
-            height="100%"
+  <card v-if="true" outlined-body flat>
```

Comment:
remove the if

Reply:
A yes that was from testing the other without removing the old component, good catch.